client/src/main/java/io/cloudsoft/winrm4j/client/ShellCommand.java

```diff
@@ -46,14 +47,21 @@
     private SelectorSetType shellSelector;
 
     private String operationTimeout;
+    /**
+     * Define if a new Receive request will be send when the server returns a fault with the code
+     * {@link #WSMAN_FAULT_CODE_OPERATION_TIMEOUT_EXPIRED}.
+     */
+    private Predicate<String> retryReceiveAfterOperationTimeout;
```

Comment:
I'd make this final (follows the same pattern as other fields, and easier to reason about the code when troubleshooting).